### PR TITLE
New ladder 12.20 updated and Tenebris spells and lever fix

### PR DESCRIPTION
**Pull request by Sargitarium**
New ladder of 12.20 update on script.

### CODE REVIEW

File: data/actions/scripts/quests/forgotten knowledge/tenebrisLever.lua [hunk 1]
```diff
@@ -1,47 +1,72 @@
...
 }
 
+local function clearTenebris()
+	local spectators = Game.getSpectators(config.bossPos, false, false, 15, 15, 15, 15)
+	for i = 1, #spectators do
+		local spectator = spectators[i]
+		if spectator:isPlayer() then
+			spectator:teleportTo(Position(32902, 31629, 14))
+			spectator:getPosition():sendMagicEffect(CONST_ME_TELEPORT)
+			spectator:say('Time out NOOB! You are teleported out by your incapacity to kill Lady Tenebris.', TALKTYPE_MONSTER_SAY)
```
Comment (gpedro) [inline data/actions/scripts/quests/forgotten knowledge/tenebrisLever.lua]: this message is real on cip server?

**Comment by Sargitarium** [comment]
A mensagem não, foi mal pelo NOOB....

Em sex, 6 de set de 2019 às 17:48, Gabriel Pedro <notifications@github.com>
escreveu:

File: data/actions/scripts/quests/forgotten knowledge/tenebrisLever.lua (hunk 1 above)
Comment (Sargitarium) [reply data/actions/scripts/quests/forgotten knowledge/tenebrisLever.lua]: Time out! You were teleported out by strange forces.

This is a correctly message for it...

Comment (Sargitarium) [reply data/actions/scripts/quests/forgotten knowledge/tenebrisLever.lua]: Edited and save....

**Pull request merged**